lib/srv/desktop/tdp/proto.go

```diff
@@ -1418,6 +1481,18 @@ func writeUint64(b *bytes.Buffer, v uint64) {
 	b.WriteByte(byte(v))
 }
 
+// tdpMaxNotificationMessageLength is somewhat arbitrary, as it is only sent *to*
+// the browser (Teleport never receives this message, so won't be decoding it)
+const tdpMaxNotificationMessageLength = 10240
+
+// TODO: tdpMaxPathLength is currently just an arbitrary value for the maximum length of
```

Comment:
Can we resolve this prior to merging to avoid creating more follow up work?

Reply:
This is an existing issue (https://github.com/gravitational/teleport/issues/14950 created July 27). May as well just figure it out here though, since otherwise it seems like one of those things that will just keep being de-prioritized.

Reply:
https://github.com/gravitational/teleport/issues/14950#issuecomment-1341632465